src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
try to use only one filter method(it`s increases perfomance), create filter function and use this in your filter method, your function should check if person is men and if century is passed check this man century and return true or false.And use better naming  
```suggestion
    people.filter(person => person.sex === 'm' && Math.ceil(person.died / 100) === century)
```